client/plots/geneset.js

```diff
@@ -0,0 +1,150 @@
+import { getCompInit, copyMerge } from '../rx'
+import { showGenesetEdit } from '../dom/genesetEdit.ts'
+import { fillTermWrapper } from '#termsetting'
+import { dofetch3 } from '#common/dofetch'
+
+const gdcGenome = 'hg38'
```

Comment:
if this plot is hardcoded for gdc, then should change file name to `gdcGeneset.js`
or if this plot will work for non-gdc dataset too?

Reply:
I pushed a commit to not hardcode.